authorization/service_test.go

```diff
@@ -63,5 +64,10 @@ func initAuthService(s kv.Store, f influxdbtesting.AuthorizationFields, t *testi
 
 func TestBoltAuthService(t *testing.T) {
 	t.Parallel()
-	influxdbtesting.AuthorizationService(initBoltAuthService, t)
+	for _, useHashedTokens := range []bool{true} {
```

Comment:
Why are you looping over a single element here?

Reply:
Good catch! `false` is also supposed to be in there.